### ISSUE TITLE
`deno run -A npm:create-hono my-app` hangs indefinitely

### ISSUE THREAD

**Issue by WrathZA** [honojs/create-hono#21]
`deno run -A npm:create-hono my-app` hangs indefinitely when attempting to create `deno` on windows and git-bash

```sh
$ deno --version
deno 1.41.1 (release, x86_64-pc-windows-msvc)
v8 12.1.285.27
typescript 5.3.3
```

`deno run -A -L debug npm:create-hono test-app >> test-app-debug.log 2>&1` 

```log
DEBUG RS - deno::args::package_json:146 - No package.json file found
DEBUG RS - deno::cache::cache_db:130 - Opening cache C:\Users\snip\AppData\Local\deno\dep_analysis_cache_v1...
DEBUG RS - deno::cache::cache_db:130 - Opening cache C:\Users\snip\AppData\Local\deno\node_analysis_cache_v1...
DEBUG RS - deno_npm::resolution::graph:972 - <package-req> - Resolved create-hono@* to create-hono@0.5.0
DEBUG RS - deno::npm::managed:301 - Resolved package folder of create-hono@0.5.0 to C:\Users\snip\AppData\Local\deno\npm\registry.npmjs.org\create-hono\0.5.0
DEBUG RS - deno::js:10 - Deno isolate init with snapshots.
DEBUG RS - deno::worker:178 - main_module file:///C:/Users/snip/AppData/Local/deno/npm/registry.npmjs.org/create-hono/0.5.0/bin
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "FORCE_COLOR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NODE_DISABLE_COLORS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NO_COLOR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TERM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "FORCE_COLOR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NODE_DISABLE_COLORS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TERM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "GRACEFUL_FS_PLATFORM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <CWD>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TEST_GRACEFUL_FS_GLOBAL_PATCH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "GRACEFUL_FS_PLATFORM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TEST_GRACEFUL_FS_GLOBAL_PATCH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TESTING_TAR_FAKE_PLATFORM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "__TESTING_MKDIRP_PLATFORM__"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "__TESTING_MKDIRP_NODE_VERSION__"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TESTING_TAR_FAKE_PLATFORM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "__FAKE_PLATFORM__"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TESTING_TAR_FAKE_PLATFORM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NO_COLOR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "FORCE_COLOR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "FORCE_COLOR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted read access to <main_module>[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "INFLUX_TOKEN"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "SSH_ASKPASS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUDRAWHWND"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "EXEPATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUPID"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "SHELL"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ACLOCAL_PATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ALLUSERSPROFILE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUANSILOG"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "HOME"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MINGW_PREFIX"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUHOOKS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "APPDATA"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "COMPUTERNAME"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUDIR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CHOCOLATEYINSTALL"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "IGCCSVC_DB"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "HOMEDRIVE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUWORKDIR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "LANG"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ORIGINAL_PATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PS1"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PATHEXT"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONFIG_SITE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PSMODULEPATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ANSICON"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUBASEDIR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NVM_SYMLINK"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PWD"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "COMMONPROGRAMW6432"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MSYSTEM_CHOST"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "INFOPATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NVM_HOME"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ORIGINAL_TEMP"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ORIGINAL_TMP"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROGRAMW6432"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "SHLVL"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TMP"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "USERNAME"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "USERPROFILE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "WINDIR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "OLDPWD"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROCESSOR_IDENTIFIER"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUISADMIN"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TERM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MANPATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUSERVERPID"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ONEDRIVE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "HOSTNAME"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MINGW_PACKAGE_PREFIX"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "SYSTEMROOT"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "USERDNSDOMAIN"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "__COMPAT_LAYER"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "OS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUDRIVE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "__PSLOCKDOWNPOLICY"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROCESSOR_LEVEL"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "COMMONPROGRAMFILES"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUBACKHWND"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "LOGONSERVER"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MSYSTEM"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TEMP"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUWORKDRIVE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "_"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "AWS_PROFILE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUBUILD"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ANSICON_DEF"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUPALETTE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "DISPLAY"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROGRAMDATA"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "SYSTEMDRIVE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NUMBER_OF_PROCESSORS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "LOCALAPPDATA"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "USERDOMAIN_ROAMINGPROFILE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "COMMONPROGRAMFILES(X86)"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "ZES_ENABLE_SYSMAN"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MINGW_CHOST"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUHWND"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MSYSTEM_PREFIX"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PLINK_PROTOCOL"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TMPDIR"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROGRAMFILES"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROCESSOR_ARCHITECTURE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROGRAMFILES(X86)"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUTASK"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "USERDOMAIN"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUBASEDIRSHORT"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CONEMUANSI"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "DRIVERDATA"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "MSYSTEM_CARCH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PKG_CONFIG_PATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "COMSPEC"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "CHOCOLATEYLASTPATHUPDATE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PUBLIC"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "HOMEPATH"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "PROCESSOR_REVISION"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "DEBUG"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "USERPROFILE"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "YARGS_MIN_NODE_VERSION"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NPM_CONFIG_USER_AGENT"[0m

create-hono version 0.5.0
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NODE_EXTRA_CA_CERTS"[0m
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted net access to "api.github.com"[0m
DEBUG RS - reqwest::connect:429 - starting new connection: https://api.github.com/
✔ Using target directory … test-app
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TERM"[0m
```

`$ deno run -A -L trace npm:create-hono test-app >> test-app.log > 2>&1`

```log

***  snipped ***

DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NPM_CONFIG_USER_AGENT"[0m
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`

create-hono version 0.5.0
TRACE RS - deno_core::gotham_state:23 -  inserting record to state for type_id `TypeId { t: 119848926350670753966736605516306532089 }`
TRACE RS - deno_core::gotham_state:60 -  taking ownership from state data for type_id `TypeId { t: 119848926350670753966736605516306532089 }`
TRACE RS - deno_core::gotham_state:48 -  mutably borrowing state data for type_id `TypeId { t: 230825253581568513391698317603144386656 }`
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "NODE_EXTRA_CA_CERTS"[0m
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 267185089902950404885088367025246449813 }`
TRACE RS - deno_core::gotham_state:48 -  mutably borrowing state data for type_id `TypeId { t: 230825253581568513391698317603144386656 }`
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted net access to "api.github.com"[0m
DEBUG RS - reqwest::connect:429 - starting new connection: https://api.github.com/
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
TRACE RS - deno_core::gotham_state:36 -  borrowing state data for type_id `TypeId { t: 210346909104299345514735826844371007692 }`
✔ Using target directory … test-app
TRACE RS - deno_core::gotham_state:48 -  mutably borrowing state data for type_id `TypeId { t: 230825253581568513391698317603144386656 }`
DEBUG RS - deno_runtime::permissions:86 - [0m[1m⚠️️  Granted env access to "TERM"[0m
TRACE RS - deno::cache::cache_db:102 - Cleaned up SQLite connection at C:\Users\brett.miller\AppData\Local\deno\node_analysis_cache_v1
TRACE RS - deno::cache::cache_db:102 - Cleaned up SQLite connection at C:\Users\brett.miller\AppData\Local\deno\dep_analysis_cache_v1

```

Let me know if I can provide anything else useful.

**Comment by ryuapp** [comment]
CLI does not seem to work properly with `deno run`.
From the second option ,we lose control of our terminal.

**Comment by yusukebe** [comment]
Is this a Windows-specific problem? The problem does not occur on my mac.

**Comment by ryuapp** [comment]
> Is this a Windows-specific problem? The problem does not occur on my mac.

That's correct. It doesn't work on Windows, but it works on WSL.

**Comment by ryuapp** [comment]
From what I've tested, this issue has probably been resolved since v0.6.1.

**Comment by yusukebe** [comment]
Greeeeat!
